tools/rapids-get-pr-wheel-artifact

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Echo path to an artifact for a specific PR. Finds and uses the latest commit on the PR.
+#
+# Positional Arguments:
+#   1) repo name
+#   2) PR number
+#   3) "cpp" or "python", to get the artifact for the C++ or Python build, respectively
+#   4) [optional] commit hash, to get the artifact for a specific commit
+#
+# Example Usage:
+#   rapids-get-pr-conda-artifact rmm 1095 cpp
+set -euo pipefail
+
+repo="$1"
+pr="$2"
+
+pkg_type="$3"
+case "${pkg_type}" in
+  cpp)
+    artifact_name=$(RAPIDS_REPOSITORY=$repo rapids-package-name wheel_cpp)
+    ;;
+  python)
+    artifact_name=$(RAPIDS_REPOSITORY=$repo rapids-package-name wheel_python)
+    ;;
+  *)
+    echo "Error: 3rd argument must be 'cpp' or 'python'"
+    exit 1
+    ;;
+esac
+
+commit="${4:-}"
+if [[ -z "${commit}" ]]; then
+    commit=$(git ls-remote https://github.com/rapidsai/"${repo}".git refs/heads/pull-request/"${pr}" | cut -c1-7)
```

Comment:
> @ajschmidt8 does CI have the necessary credentials to make this request over SSH rather than HTTPS?

Nope. Good catch. Private repo failures will have to be handled appropriately here.

Reply:
Maybe we should do that for the conda tool as well in this PR then to keep things aligned.

Reply:
This is copied frrom the conda tool. I'd like to separate this issue of private repo support in this command from the effort of adding equivalency between conda and wheels.

Would it be OK with you (plural) to punt on this?

Reply:
Yup, punt away

Reply:
https://github.com/rapidsai/gha-tools/issues/103